docs/CSS28-migration_guide.md

```diff
@@ -0,0 +1,144 @@
+# Migration Guide
+
+[Implementers’ doc] [WIP]
+
+This document aims to list changes that were made in `v.2` and how to handle them in your Reading System/apps when upgrading ReadiumCSS. 
+
+## Removal of responsive columns
+
+In version 1, ReadiumCSS used media queries to automatically handle the number of columns/pages to display based on certains conditions. To sum things up it switched from 1 to 2 columns when:
+
+- the `width` of the window was large enough, based on the font-size;
+- the device’s width (`device-width`) was within an arbitrary range based on the font-size, and in landscape orientation.
+
+As a side-effect, this “auto-pagination model” (think of it as an “auto” setting for the number of columns/pages in user settings) was [not reliable enough for implementers](https://github.com/readium/readium-css/issues/143), and created issues with newer devices and form-factors (e.g. foldables). Sometimes it wouldn’t even swith to 2 columns when applying the setting because of the conditions above.
+
+In version 2, ReadiumCSS removed these media queries and the “auto-pagination model” entirely. It applies styles the Reading System/app provides to it. Consequently, control over breakpoints is now the responsibility of the app. 
+
+If you want to reimplement an “auto” user setting, and switch to 1 or 2 columns depending on the orientation of the device or the width of the window, you now have to handle it programmatically.
+
+## Number of columns is no longer limited
+
+In version 1, ReadiumCSS limited the number of columns to `1` or `2`.
+
+In version 2, the number of columns is no longer limited. Please note value `0` is handled as an error and resolves to `1` though.
+
+## Addition of a flag to disable vertical-writing pagination
+
+Given the limitations of columns fragmentation in vertical writing – progression direction from top to bottom instead of right to left, columns stacked on the `y-axis` –, some Reading Systems and apps have been disabling it, and implement their own logic.
```

Comment:
You can think of it as a mere alias to scroll mode – it’s actually re-using the exact same declaration. It was just added for convenience, in case it makes more sense to other implementers to use this explicit flag instead of switching to scroll.

Reply:
Alright, I think we will stick to the scroll mode on mobile as we have to show it in the user preferences anyway.